docker-compose.yaml

```diff
@@ -50,6 +50,8 @@ services:
       dockerfile: ./Dockerfile
       args:
         FE_BE_HOST: ${BE_HOST:-$DOMAIN:5004}
+        JELLYFISH_VERSION: ${JELLYFISH_VERSION:-0.2.0}
+        JELLYROOM_VERSION: ${JELLYROOM_VERSION:-build}
```

Comment:
"build" is just a placeholder to show on FE, looking a bit better than "undefined"